### PR TITLE
CAT: changing format or airbyte_type is a breaking change

### PR DESCRIPTION
**Pull request by davydov-d**
## What
https://github.com/airbytehq/airbyte/issues/30096

## How
- Fail CAT if a `format` or an `airbyte_type` is changed in a field definition of either a spec or a schema

### CODE REVIEW

File: airbyte-integrations/bases/connector-acceptance-test/Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN poetry install --no-root --only main --no-interaction --no-ansi
 COPY . /app
 RUN poetry install --only main --no-cache --no-interaction --no-ansi
 
-LABEL io.airbyte.version=2.0.0
+LABEL io.airbyte.version=2.0.1
```
Comment (alafanechere) [inline airbyte-integrations/bases/connector-acceptance-test/Dockerfile:27]: Please also bump the version in `pyproject.toml`

**Review by alafanechere — APPROVED** [review]
Nice! Thank you.
Please bump the version in the pyproject.toml file of CAT.
Let me know when you want to publish CAT, the workflow is a bit broken ATM.

**Comment by davydov-d** [comment]
@alafanechere thanks, I have bumped version in pyproject.toml. I'd like to ship it as soon as possible please

**Comment by alafanechere** [comment]
> @alafanechere thanks, I have bumped the version in pyproject.toml. I'd like to ship it as soon as possible please

Feel free to merge, I'll publish later. The CI will use your version (we just need to publish for /legacy-test).

**Pull request merged**